### ISSUE TITLE
Gzip/Brotli compression support?

### ISSUE THREAD

**Issue by EricBlanchard** [elraccoone/react-unity-webgl#120]
Firstly love the package! I have it up and running, working beautifully, but I'm running into some pretty large initial load times for unity project. My web server is setup using nginx and serves up Application/wasm as wasm, but I'm only able to get it to work currently with uncompressed builds. I understand I'll have to probably mess around with some Nginx settings as well, but is there anything specific I need to from the react side to get it to use the compressed files correctly? Ideally wasm streaming from gzip/brotli? Am I missing something obvious? 

Thanks friends!

Eric

**Comment by jeffreylanters** [comment]
Hi Eric! These are server side settings. You can configure NGINX to serve these kind of files. Do you have experience with NGINX configuration?

**Comment by EricBlanchard** [comment]
Thanks for the response, I don't have any prior nginx experience before doing this project. I've enabled gzip on the nginx side, but I think what's happening right now is it's taking the uncompressed file, compressing it with gzip, firing it over and then react is consuming it. Ideally I'd want to serve the ALREADY compressed .gz files. I believe I need to set that up still on the NGINX side, but I wanted to double check if there was anything I needed to also do on the react side. But it sounds like it's just server side changes I need to make?

**Comment by jeffreylanters** [comment]
No, there is nothing you have to do on the client side. These settings should be configured on the server side, when using a Linux based system you can follow [these steps.](https://docs.nginx.com/nginx/admin-guide/dynamic-modules/brotli/)

**Comment by jeffreylanters** [comment]
Best of luck!

**Comment by motiondirector** [comment]
With gzip enabled on server side, can I simply use .gz files in UnityContext? 

const WEBGL_BUILD = 'my2020webGLbuild';
...
    unityContext: new UnityContext({
        loaderUrl: `./${WEBGL_BUILD}/Build/${WEBGL_BUILD}.loader.js`,
        dataUrl: `./${WEBGL_BUILD}/Build/${WEBGL_BUILD}.data.gz`,
        frameworkUrl: `./${WEBGL_BUILD}/Build/${WEBGL_BUILD}.framework.js.gz`,
        codeUrl: `./${WEBGL_BUILD}/Build/${WEBGL_BUILD}.wasm.gz`
    }),
...

**Comment by jeffreylanters** [comment]
Yes, @motiondirector
